res/css/photon-components.css

```diff
@@ -121,3 +154,139 @@
 .photon-message-bar-error .photon-button:hover:active {
   background-color: var(--red-90);
 }
+
+/* Label */
+
+.photon-label,
+.photon-label-micro {
+  display: inline-flex;
```

Comment:
The small caveat here is that if we want a "block" behavior we'd need to know we should use `display: flex` as override. The only solution I see is to have an extra block around, so my suggestion is to keep it like this. I mentioned it in case you have a better idea.